docs/tutorial/spellchecker.md

```diff
@@ -0,0 +1,74 @@
+# SpellChecker
+
+Electron has built-in support for Chromium's spellchecker since Electron 8.  On Windows and Linux this is powered by Hunspell dictionaries, on macOS it makes use of the native spellchecker APIs.
+
+## How to enable the spellchecker?
+
+For Electron 9 and higher the spellchecker is enabled by default.  For Electron 8 you need to enable it in `webPreferences`.
+
+```js
+const myWindow = new BrowserWindow({
+  webPreferences: {
+    spellcheck: true
+  }
+})
+```
+
+## How to set the languages the spellchecker uses?
+
+On macOS as we use the native APIs there is no way to set the language that the spellchecker uses, by default on macOS the native spellchecker will automatically detect the language being used for you.
```

Comment:
nit(comma splice): On macOS as we use the native APIs there is no way to set the language that the spellchecker uses. By default, on macOS, the native spellchecker will automatically detect the language being used for you.